modal/volume.py

```diff
@@ -139,7 +139,9 @@ def _initialize_from_empty(self):
     def new() -> "_Volume":
         """`Volume.new` is deprecated.
 
-        Please use `Volume.from_name` (for persisted) or `Volume.ephemeral` (for ephemeral) volumes.
+        You can switch this out for `Volume.from_name("some-volume-identifier")` for persisted dicts.
+        You can also take a look at the documentation for `Volume.ephemeral` for another option,
+        although it might require more code changes if you're coming from `Volume.new`.
```

Comment:
I had a hard time finding which docs this refers to. Is it https://modal.com/docs/reference/modal.Volume#ephemeral? Ideally we should add a guide page on "ephemeral objects" and link to that here directly